snippets/saved_replies.md

```diff
@@ -24,9 +24,12 @@ It's not always feasible, or even possible, to fill out the XXX used in the snip
 
 Thanks for your report. It looks as if this is actually a question about usage and not development because XXX
 
-To make your question, and all replies, easier to find, we suggest you move this over to our [user mailing list](https://groups.google.com/forum/#!forum/prometheus-users), which you can also search. If you prefer more interactive help, join or our [IRC channel](https://webchat.freenode.net/?randomnick=1&channels=prometheus), #prometheus on irc.freenode.net. Please be aware that our IRC channel has no logs, is not searchable, and that people might not answer quickly if they are busy or asleep. If in doubt, you should choose the mailing list.
+To make your question, and all replies, easier to find, I suggest you move this over to our `[user mailing list](https://groups.google.com/forum/#!forum/prometheus-users)`.
 
-Once your questions have been answered, please add a short line pointing to relevant replies in case anyone stumbles here via a search engine in the future.
+If you haven't looked already you might find your answer in the `[official docs and examples](https://prometheus.io/docs/introduction/overview/)` or by searching in the `[users](https://groups.google.com/forum/#!forum/prometheus-users)` or `[devs](https://groups.google.com/forum/#!forum/prometheus-developers)` groups.<br/>
+The #prometheus `[IRC channel ](https://webchat.freenode.net/?channels=prometheus)` is also a great place to  mix up with the community and ask questions(don't be afraid to answers few while waiting) `:+1:`.
```

Comment:
I'd keep this focused on the user's question.

Reply:
@RichiH what do you think?